src/js/tech/tech.js

```diff
@@ -267,21 +333,15 @@ class Tech extends Component {
 
   setCurrentTime() {
     // improve the accuracy of manual timeupdates
-    if (this.manualTimeUpdates) { this.player().trigger('timeupdate'); }
+    if (this.manualTimeUpdates) { this.trigger('timeupdate'); }
   }
 
   // TODO: Consider looking at moving this into the text track display directly
   // https://github.com/videojs/video.js/issues/1863
   initTextTrackListeners() {
-    let player = this.player_;
-
-    let textTrackListChanges = function() {
-      let textTrackDisplay = player.getChild('textTrackDisplay');
-
-      if (textTrackDisplay) {
-        textTrackDisplay.updateDisplay();
```

Comment:
It looks like we're removing this but not replacing it elsewhere. We should probably do what the TODO above it says. 'Consider looking at moving this into the text track display directly'. ie. have the textTrackDisplay listen for player texttrackchange events.